engine/core/src/main/java/org/pentaho/reporting/engine/classic/core/modules/output/pageable/pdf/internal/PdfLogicalPageDrawable.java

```diff
@@ -124,6 +124,7 @@ public void close() {
   }
 
   private static final float ITALIC_ANGLE = 0.21256f;
+  private static final String UNICODE_SPACES_REGEX = "[\\u00A0\\u202F\\u2007\\u2009\\u200A\\u2000-\\u200B]";
```

Comment:
\u2007, \u2009, and \u200A is already included on the [\u2000-\u200B] interval...
Maybe change to "[\\u00A0\\u202F\\u2000-\\u200B]"?